.travis.yml

```diff
@@ -13,4 +13,11 @@ before_cache:
 cache:
   directories:
     - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
\ No newline at end of file
+    - $HOME/.gradle/wrapper/
+
+
+notifications:
+  slack:
+    on_success: always
+    on_failure: change
+    secure: WOingZJ/nk8Mc5JXbHgE5Ux1YwpsrWryft5TH8kPBaIqTmX6qlcu3KSdLktwgiPgXJiOWpR6UitX9+xfalcApEfS77ZNQeUFafDYHv17nS6IAHe/A4vOAIElXfpS2KnY0aqxwWhgKCsehio49SPY7WHq9PtnvauEdkX8bca5FHk=
```

Comment:
This is an encrypted key. This key is not the actual slack key.
This is what is added to the `.travis.yml` when you run the command:
`travis encrypt "[secure key here]" --add notifications.slack -r wpilibsuite/allwpilib